fly.toml

```
@@ -1,5 +1,8 @@
-app = "fly-log-shipper-example"
+app = "logtail-log-shipper"
```

```suggestion
app = "udisc-log-shipper"
```
name of app doesn't need to reference the vendor; since we may have multiple vendors.